### DISCUSSION TITLE
Release 2.4.9

### DISCUSSION THREAD

**Discussion by revoxhere** [revoxhere/duino-coin#569] [General]
### Hey there! This update addresses bug fixes and improvements that have been done over the past three weeks.

Changelog:
  * Fixed some reconnection issues
  * Optimized the Arduino Code (up to 20% higher hashrates!)
  * Added Raspberry Pi Pico support
  * Improved stability

### ⚠️ NOTICE: .exe binaries may be detected as a virus by your antivirus software. This is a **false positive** caused by pyinstallers bootloader, read about it [here](https://stackoverflow.com/questions/43777106/program-made-with-pyinstaller-now-seen-as-a-trojan-horse-by-avg). Duino-Coin is not a virus.

For the next release (2.5) we're planning to ship the pool system. We know it's a bit delayed, but it needs some more work.
Anyway, we promise that it'll be out soon.
Happy mining!

<hr /><em>This discussion was created from the release <a href='https://github.com/revoxhere/duino-coin/releases/tag/2.4.9'>Release 2.4.9</a>.</em>

**Comment by KnowTheFuture** [comment]
@revoxhere I saw your message the other after the update about taking a needing to take a break. I'm more than 2x your age, and when I was a kid all wanted to do was learn how to program. Unfortunately, my family did not support my desire to learn.

When I was 20 years old, I worked for a sales company as a branch manager. I thought I was a big shot, running my own office at the age of 20. It failed miserably, and I dealt with depression and illness afterwards. (I felt like a failure, and I drank about 3L of Mountain Dew (high-sugar soda) everyday to stack awake. Not only did it make it worse, I ended up dealing with kidney stones.

I think you are learning some very important skills, such as the importance of taking breaks, but I would encourage you not to give up now. You're doing something that many cannot do for themselves.  For every negative comment, there are 10 good ones, it's just the bad ones such out and hurt more.

Don't give up!

**Reply by revoxhere** [reply]
Thanks for these words, I'll keep them in mind. I'm definitely not giving up

**Comment by PakyITA** [comment]
I have a problem on my raspberry pi 3 b+ when i star PC_Miner.py i have connection error: [Photo](https://imgur.com/a/wG6vwlD)

**Comment by primitt** [comment]
> I have a problem on my raspberry pi 3 b+ when i star PC_Miner.py i have connection error: [Photo](https://imgur.com/a/wG6vwlD) 

PC Mining is currently disabled due to server overloading, more info on the duinocoin discord server. (https://discord.gg/CybTZ66J)

**Comment by harinandhan15** [comment]
When will be pc mining resume ? Please allow pc mining ASAP

**Comment by primitt** [comment]
> When will be pc mining resume ? Please allow pc mining ASAP

It will be enabled when pools come out. More info on the discord.

**Comment by harinandhan15** [comment]
When we will get 2.5 for pool mining?

**Comment by primitt** [comment]
> When we will get 2.5 for pool mining?

No ETA as for now. Maybe more than 1 week.

**Comment by Harveyn4444** [comment]
How do I update my miner on raspberry pi, I have error messages saying my miner is outdated (V 2.45) how can I update my miner to match the server 
Thanks :)

**Reply by Harveyn4444** [reply]
it should be said that i'm using raspbian lite so I don't know the commands to update the miner

**Reply by revoxhere** [reply]
git pull

**Reply by Harveyn4444** [reply]
Where/how do I do this command

**Reply by revoxhere** [reply]
In the terminal while being in the duino-coin directory
